langcodes/db.py

```diff
@@ -152,7 +152,12 @@ def add_language(self, data, datalang, name_order=0):
             'language',
             (subtag, script, is_macro, is_collection, preferred, macrolang)
         )
-        if not 'Preferred-Value' in data:
+        if 'Preferred-Value' in data:
+            desc = ';'.join(data.get('Description', []))
```

Comment:
I'm not sure that I see what's going on with this addition.  Things in the data file with a "Preferred-Value" have descriptions; but what would make them lists that need to be joined with a semicolon? Does it make sense to have one more line in the doc test to illustrate a case where this happens?

Incidentally and separately: above this line is `preferred = data.get('Preferred-Value')`.  That probably means you can use `if preferred` (or at least `if preferred is not None`) as the conditional, and you can certainly replace `data['Preferred-Value']` with `preferred` just below this.